style.css

```diff
@@ -1 +1,193 @@
 /*  Add your styling here */
+body {
+  font-family: "Roboto", sans-serif;
+  width: 100%;
+  margin:0px;
+}
+
+.navbar {
+  display: none;
+}
+.nav {
+  display: none;
+}
+header {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  background-color: #7b3f00;
+ 
+}
+.header {
+  width: 60%;
+}
+.sideimg {
+  width: 40%;
+}
+.headerimg {
+  width: 150px;
+  padding: 25px 0px 35px 25px;
+}
+.hamburger {
+  width: 30px;
+  position: absolute;
+  top: 30px;
+  right: 30px;
+}
+.sitemoto {
+  text-align: right;
+  padding: 100px 30px 0px 25px;
+  font-size: 1.2rem;
+  color: lightgoldenrodyellow;
+}
+img {
+  transition: 1sec;
+}
+img:hover {
+  transform: scale(0.97);
+}
+
+section {
+  padding: 10px;
+}
+.maintext1 {
```

Comment:
A better name for this would be "title" or "welcome-msg" - try to be more descriptive.